hydra-editor.gemspec

```diff
@@ -19,15 +19,14 @@ Gem::Specification.new do |s|
   s.add_dependency "active-fedora", ">= 9.0.0"
   s.add_dependency "almond-rails", '~> 0.1'
   s.add_dependency "cancancan", "~> 1.8"
-  s.add_dependency "rails", ">= 5.0.7", "< 6"
-  s.add_dependency "simple_form", '~> 4.0'
+  s.add_dependency "rails", ">= 5", "< 6"
+  s.add_dependency "simple_form", '~> 4.1.0'
```

Comment:
I'm going to defer to @no-reply for this, as I'm still finding this to be a later release than the warning on https://github.com/samvera/hyrax/blob/master/hyrax.gemspec#L116

Reply:
This was discussed on Slack, and it was clarified that this should not block the proposed release.